tensorflow_addons/metrics/r_square.py

```diff
@@ -51,7 +51,7 @@ def __init__(self, name='r_square', dtype=tf.float32):
         self.res = self.add_weight("residual", initializer="zeros")
         self.count = self.add_weight("count", initializer="zeros")
 
-    def update_state(self, y_true, y_pred):
+    def update_state(self, y_true, y_pred, sample_weight=None):
```

Comment:
I believe `sample_weight` needs to be applied? See https://github.com/tensorflow/tensorflow/blob/r2.0/tensorflow/python/keras/metrics.py#L317-L329

Reply:
For now, we are adding it to be done. Will create a new PR with implementation for the same.